docs/src/managing-packages.md

```diff
@@ -208,6 +208,42 @@ For example, the tracked dependencies can be stored inside of the active project
 The whole directory can be moved and `Pkg` will still be able to find the dependencies
 because their path relative to the active project is preserved even though their absolute path has changed.
 
+### Adding a subdir package
```

Comment:
```suggestion
### Adding a package in a subdirectory of a repository
```

or something? "subdir" is short in the API but I think it can be allowed to be fully spelled out in a documentation heading.